packages/devtools_app_shared/lib/src/ui/theme/theme.dart

```diff
@@ -599,3 +603,23 @@ ButtonStyle _generateButtonStyle({
   }
   return buttonStyle;
 }
+
+extension on Color {
+  Color darken({int percent = 10}) {
+    assert(1 <= percent && percent <= 100);
+    final p = percent / 100;
```

Comment:
can we make these methods take a value between 0.0 and 1.0 instead of having to / 100 to get the percentage? This is similar to flutter's opacity APIs for example.